### PR TITLE
[MIG][15.0] stock_account: migration script

### PR DESCRIPTION
**Pull request by sang250399**
This PR
======
Migration done

### CODE REVIEW

**Review by gdgellatly — COMMENTED** [review]
Failing on trying to add default/demo warehouse

File: openupgrade_scripts/scripts/stock_account/15.0.1.1/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+NEW ir.ui.view: stock_account.stock_account_report_invoice_document
+NEW ir.ui.view: stock_account.stock_inventory_request_count_form_view_inherit_stock_account
+NEW ir.ui.view: stock_account.stock_valuation_layer_pivot
+NEW ir.ui.view: stock_account.view_category_property_form_stock
+NEW ir.ui.view: stock_account.view_stock_quant_tree_inventory_editable_inherit_stock_account
+DEL ir.ui.view: stock_account.view_inventory_form_inherit
+DEL ir.ui.view: stock_account.view_inventory_tree
+# NOTHING TO DO: noupdate="0" records
+
+NEW res.groups: stock_account.group_lot_on_invoice [renamed from sale_stock module]
```
Comment (hieu1211) [inline openupgrade_scripts/scripts/stock_account/15.0.1.1/upgrade_analysis_work.txt]: You may have forgotten to change noupdate="1"
https://github.com/OCA/OpenUpgrade/blob/15.0/openupgrade_scripts/scripts/stock_account/15.0.1.1/upgrade_analysis.txt#L14

**Comment by sang250399** [comment]
> Failing on trying to add default/demo warehouse

The reason is that the stock module has not been merged

**Review by phamgiang2510 — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/OpenUpgrade/pull/3527.

**Pull request closed without merging**